server/pom.xml

```diff
@@ -33,10 +33,10 @@
             <groupId>jakarta.websocket</groupId>
             <artifactId>jakarta.websocket-api</artifactId>
         </dependency>
-        <dependency>
+        <!--dependency>
```

Comment:
This is probably a transitive dependency of the library below and should be uncommented again